zim/plugins/pageindex/__init__.py

```diff
@@ -539,7 +539,7 @@ def do_drag_data_received(self, dragcontext, x, y, selectiondata, info, time):
 		assert selectiondata.get_target().name() == PAGELIST_TARGET_NAME
 		data = selectiondata.get_data()
 		logger.debug('Drag data recieved: %r', data)
-		if data is None:
+		if data is None or len(data)==0:
```

Comment:
Please adopt the coding style as it is present in the file, i.e. add spaces. See here for example: https://github.com/zim-desktop-wiki/zim-desktop-wiki/blob/26de261b13f1560e315bb6914e0f91c61ed38ac0/zim/plugins/pageindex/__init__.py#L580